sonar-project.properties

```diff
@@ -6,7 +6,7 @@ sonar.project.monorepo.enabled=true
 
 sonar.sources=.
 sonar.exclusions=**/*_test.go,**/*.pb.go,**/*.pulsar.go,**/*.pb.gw.go
```

Comment:
can we add *.java exclusion? https://github.com/cosmos/cosmos-sdk/actions/runs/9323449491/job/25666860500?pr=20488